roadmap/implementers-guide/src/node/backing/statement-distribution.md

```diff
@@ -1,107 +1,453 @@
+        -   Runs in a background responder task which feeds requests to `answer_request`
+            through `MuxedMessage`.
+        -   This responder task has a limit on the number of parallel requests.
+    -   `answer_request` on the peer takes the request and sends a response.
+        -   Does this using the response sender on the request.
+
+3.  Requesting Validator
+
+    -   `receive_response` on the original validator yields a response.
+        -   Response was sent on the request's response sender.
+        -   Uses `RequestManager::await_incoming` to await on pending responses in an
+            unordered fashion.
+        -   Runs on the `MuxedMessage` receiver.
+    -   `handle_response` handles the response.
+
+### API
+
+-   `dispatch_requests`
+    -   Dispatches pending requests for candidate data & statements.
+-   `answer_request`
+    -   Answers an incoming request for a candidate.
+    -   Takes an incoming `AttestedCandidateRequest`.
+-   `receive_response`
+    -   Wait on the next incoming response.
+    -   If there are no requests pending, this future never resolves.
+    -   Returns `UnhandledResponse`
+-   `handle_response`
+    -   Handles an incoming response.
+    -   Takes `UnhandledResponse`
+
+## Manifest
+
+-   What it is: A message about a known backed candidate, along with a description
+    of the statements backing it.
+-   Can be one of two kinds:
+    -   `Full`
+        -   Contains information about the candidate and should be sent to peers who
+            may not have the candidate yet.
+    -   `Acknowledgement`
+        -   Omit information implicit in the candidate, and should be sent to peers
+            which are guaranteed to have the candidate already.
+
+### Manifest exchange
+
+-   Occurs between us and a peer in the grid.
+-   Why:
+    -   Indicates that both nodes know the candidate as valid and backed.
+    -   Allows the nodes to send `Statement` messages directly to each other for any
+        new statements they might need.
+-   Both `manifest_sent_to` and `manifest_received_from` have been invoked.
+-   In practice, it means that one of three things have happened:
+    -   They announced, we acknowledged
+    -   We announced, they acknowledged
+    -   We announced, they announced (not sure if this can actually happen; it would
+        happen if 2 nodes had each other in their sending set and they sent
+        manifests at the same time. The code accounts for this anyway)
+-   After conclusion, we update pending statements.
+    -   We now know our statements and theirs.
+    -   Pending statements are those we know locally that the remote node does not.
+
+## Cluster
+
+-   Direct distribution of unbacked candidates within a group.
+-   Why:
+    -   because of prospective parachains communication has to happen between
+        backing groups on rotation boundaries.
+    -   Only have to send unbacked candidates within groups.
```

Comment:
It's my first time seeing "unbacked". Don't we just call it "seconded"?

Reply:
hm, maybe it's not used elsewhere. it seems strictly more correct than "seconded", though, in the sense that backed candidates are also seconded, even though not all seconded candidates are backed.

Reply:
I figured no clarification here was necessary so marking as resolved. Feel free to suggest changes, though.